p2p/simulations/http.go

```diff
@@ -260,6 +272,8 @@ func NewServer(config *ServerConfig) *Server {
 	s.GET("/networks", s.GetNetworks)
 	s.GET("/networks/:netid", s.GetNetwork)
 	s.GET("/networks/:netid/events", s.StreamNetworkEvents)
+	s.POST("/networks/:netid/snapshots/create", s.CreateSnapshot)
```

Comment:
Cmon man, when I search google, I create a search right? no the resources there are networks, nodes and connections, we dont create or change anything there. A snapshot is actually the same as an update or index. 
In fact it does not even change any state on the network so it is a very clear case for GET

for load, i disagree as well, what you quote for PUT is the purist PUT, everybody uses PUT as PATCH basically, ie., update partial info on an existing resource. or change its state.
That is what we are doing.

POST a snapshot is also possible but it should mean that the network itself is created in the very same step. But on second thought that is probably the more common usecase but then the action is not `loadSnapshot` but `CreateNetworkFromSnapshot`